pulpcore/cli/ansible/content.py

```diff
@@ -27,5 +35,60 @@ def content(ctx: click.Context, pulp_ctx: PulpContext, content_type: str) -> Non
     else:
         raise NotImplementedError()
 
+list_options = [
+    pulp_option("--name", help=_("Name of {entity}")),
+    pulp_option("--namespace", help=_("Namespace of {entity}")),
+    pulp_option("--version", help=_("Version of {entity}")),
+    # pulp_option("--only-highest", "is_highest", is_flag=True, help=_("Only show highest version of collection version (only works for collection versions)")),
+    pulp_option("--tags", help=_("Comma separated list of tags that must all match (only works for collection versions)")),
+]
+
+fields_options = [
+    pulp_option("--fields", help=_("String list of fields to include in the result")),
+    pulp_option("--exclude-fields", default="files", help=_("String list of fields to exclude from result"))
+]
+
+lookup_options = [
+    click.option("--name",  help=_("Name of {entity}"), group=["namespace", "version"], expose_value=False, cls=GroupOption, callback=_content_callback),
+    click.option("--namespace",  help=_("Namespace of {entity}"), group=["name", "version"], expose_value=False, cls=GroupOption),
+    click.option("--version",  help=_("Version of {entity}"), group=["namespace", "name"], expose_value=False, cls=GroupOption),
+    href_option,
+]
+
+content.add_command(list_command(decorators=list_options + fields_options))
+content.add_command(show_command(decorators=lookup_options))
+
 
-content.add_command(list_command())
+@content.command()
+@click.option("--file", type=click.File("rb"), required=True)
+@click.option(
```

Comment:
Don't we have a generic for this?

Reply:
I mean the chunk-size

Reply:
Yes, I didn't rebase when making this so copying pasting made me use the old option.